_features/installable.html

```diff
@@ -2,8 +2,17 @@
 title: Installable
 slug: installable
 description: Keep user's most useful app on their home screen without the hassle of an app store.
+usecases:
+- As a user, I want my favourite web applications (GMail, YouTube,
+  Facebook&hellip;) to appear in my homescree as if they were been installed from
+  the marketplace.
+- As a browser developer, I want to offer the users the posibility of provide a
+  native user experience for a web site.
 technologies:
 - web-manifest
-- service-workers
 ---
-<p>Progressive Web Apps can work even on lack of network or unreliable connectivity. No more blank connectivity error pages nor dinosaurs running through the desert. A clear separation between UI and content in addition to offline caches and service workers allow you to store the application layout for future use.</p>
+<p>You know there are some few webs you use so much that you would want them to be
+among your apps, there in your homescreen. Not a bookmark because
+you don't want them to be open in the browser but as if they were real native
+applications. This is what you get with Progressive Web Applications almost
+by free.</p>
```

Comment:
Nits:
* webs -> web apps
* in your homescreen -> on your homescreen
* by free -> for free


Also, this paragraph feels slick (in the advertising sense of the word), which will put off some readers. I wouldn't presume to tell a reader what "you know." And I wouldn't tell them that they get something "almost for free."

So I'd make this something like (with a few other trivial stylistic modifications):

> There are a few web apps you use so much that you want them to be among your native apps, there on your homescreen. Not like a bookmark, because you don't want them to open in the browser, but as if they were real native apps. This is what you get with Progressive Web Apps.